man/pdfpcrc.in

```diff
@@ -92,6 +89,13 @@ Enable color hints of the timer, continuously indicating whether the
 presentation is progressing according to the expected pace (bool, Default true).
 See the timer operation description in
 .B pdfpc(1).
+.TP
+.B toolbox-direction
+Set the toolbox direction (LtoR, RtoL, TtoB, or BtoT for the left-to-right,
+right-to-left, top-to-bottom, or botton-to-top, respectively). Default is LtoR.
+.TP
+.B toolbox-state
+Set the toolbox state (Shown|Hidden|Minimized). Default is Minimized.
```

Comment:
I would prefer to auto-hide it, or do some tablet detection.

Reply:
In fact, I think it is also usable for non-tablet hardware. Especially for new users, who have not yet memorized keyboard shortcuts. (Well, I personally use it :).) As such, if it is hidden by default, the new feature will remain unnoticed. On the other hand, if shown by default and somebody finds it annoying, it is a matter of adding a single config statement to pdfpcrc...

Reply:
OK, let's make the toolbox hidden by default. I then suggest to add toggleToolbox action and bind it, say, to "t".